### PR TITLE
Feature/footer component

### PR DESCRIPTION
**Pull request by MLucky518**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +32,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Basic SPA</title>
+    <title>Pet-XPress</title>
```
Comment (wilcoxva) [inline public/index.html:35]: I like it!

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './footer.scss';
+import logo from '../../assets/9f5a8b6a-18d6-4a28-9e98-1eee8c05b05a_200x200.png';
+
+function Footer() {
+  return (
+    <div className="footer-container">
+      <img src={logo} alt="cat and dog together" />
+      <p className="footer-name">Pet-XPress © 2020</p>
+      <div className="footer-icons">
```
Comment (wilcoxva) [inline src/components/Footer/Footer.js:10]: nice icons

File: src/components/pages/Profile/Profile.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const Profile = props => {
   useEffect(() => {
     // Check current user type to see if groomer or not
 
-    if (props.currentUser.user_type === 'groomer') {
+    if (props.currentUser.is_groomer) {
```
Comment (wilcoxva) [inline src/components/pages/Profile/Profile.js:13]: thanks for changing this

File: src/components/Footer/Footer.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        <i className="fab fa-facebook footer-icon"></i>
+        <i className="fab fa-twitter footer-icon"></i>
+        <i className="fab fa-instagram footer-icon"></i>
+      </div>
+    </div>
+  );
+}
+
+export default Footer;
```
Comment (tlewandowski18) [inline src/components/Footer/Footer.js:19]: Footer component looks solid

File: src/components/Map/MapBox.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+import { Link } from 'react-router-dom';
+import MapModal from './MapModal';
+import { Tooltip } from 'antd';
+
+const groomers = [
+  { name: 'groomer1', lat: 42.7478, lon: -73.7605 },
+  { name: 'groomer2', lat: 42.6478, lon: -73.6605 },
+  { name: 'groomer3', lat: 42.8478, lon: -73.8605 },
+  { name: 'groomer4', lat: 42.9478, lon: -73.9605 },
+];
```
Comment (tlewandowski18) [inline src/components/Map/MapBox.js:14]: Will need to update with real data

**Review by tlewandowski18 — APPROVED** [review]

**Pull request merged**